model/src/w3timemd.F90

```diff
@@ -77,6 +77,9 @@ MODULE W3TIMEMD
       INTEGER, PRIVATE        :: PRFTB(8)
       LOGICAL, PRIVATE        :: FLPROF = .FALSE.
       CHARACTER, PUBLIC       :: CALTYPE*8
+#ifdef CESMCOUPLED
+      LOGICAL, PUBLIC         :: NOLEAP = .TRUE.
```

Comment:
I think Cecilia put in the leap-year related changes to fix a bug, so not CESM specific

Reply:
@cmbitz can you confirm if this should be pulled out a bug-fix and not something CESM specific?

Reply:
I think this and the change at line 551 (below) are because the calendar type is initialized in the mod_def file instead of a namelist. I believe we can resolve this issue in the same way @mvertens resolved changing the timestep after the model initialization in the ``waveinit_cesm`` routine but we will need to test and verify this before removing this particular ifdef.

Reply:
@JessicaMeixner-NOAA The CESMCOUPLED ifdefs to hardcode noleap have been removed.